openmc/universe.py

```diff
@@ -744,35 +748,42 @@ def bounding_region(self, bounded_type='box', boundary_type='vacuum'):
 
         check_type('boundary type', boundary_type, str)
         check_value('boundary type', boundary_type, _BOUNDARY_TYPES)
+        check_type('starting surface id', starting_id, Integral)
         check_type('bounded type', bounded_type, str)
         check_value('bounded type', bounded_type, ('box', 'sphere'))
 
-        bounding_box = self.bounding_box
+        bbox = self.bounding_box
 
         if bounded_type == 'sphere':
-            import math
-            bounding_box_center = (bounding_box[0] + bounding_box[1])/2
-            radius = math.dist(bounding_box[0], bounding_box[1])
+            bbox_center = (bbox[0] + bbox[1])/2
+            radius = np.linalg.norm(np.asarray(bbox))
             bounding_surface = openmc.Sphere(
-                x0=bounding_box_center[0],
-                y0=bounding_box_center[1],
-                z0=bounding_box_center[2],
+                surface_id=starting_id,
+                x0=bbox_center[0],
+                y0=bbox_center[1],
+                z0=bbox_center[2],
                 boundary_type=boundary_type,
                 r=radius,
             )
 
             return -bounding_surface
 
         if bounded_type == 'box':
+            surf_ids = [starting_id+i for i in range(6)]
```

Comment:
I like the list comprehension used, I'm just wondering if a basic approach is easier

```suggestion
```